packages/neos-ui-views/src/NodeInfoView/index.js

```diff
@@ -28,11 +28,11 @@ export default class NodeInfoView extends PureComponent {
         const node = getNodeByContextPath(focusedNodeContextPath);
         const properties = {
             identifier: $get('identifier', node),
-            created: $get('properties._creationDateTime', node),
-            lastModification: $get('properties._lastModificationDateTime', node),
-            lastPublication: $get('properties._lastPublicationDateTime', node),
-            path: $get('properties._path', node),
-            name: $get('properties._name', node) ? $get('properties._name', node) : '/'
+            created: $get('creationDateTime', node),
+            lastModification: $get('lastModificationDateTime', node),
```

Comment:
`lastModification` AND `lastPublication` can be `null`. Therefore we need to add a check for `lastModification` in line 46 as already implemented in line 50.

Reply:
as I am doing it, I wonder, do we want to keep the row and just show nothing? Otherwise the Display jumps between nodes